content/en/docs/2024.3/Reference/data-types/http/soap/soapmessage.md

```diff
@@ -1,20 +1,69 @@
 ---
 title: "SoapMessage"
 linkTitle: "SoapMessage"
-description: "Used to represent a SOAP message."
+description: "Any data type that is used to represent a SOAP message."
 weight: 1
 ---
 
 # {{% param title %}}
 
 <p class="namespace">(Cortex.DataTypes.Http.Soap.SoapMessage)</p>
 
-{{< workinprogress >}}
+{{% alert type="information" title="Information" %}} Improvements to this page are planned for the future. {{% /alert %}}
 
 ## Summary
 
+Any data type that is used to represent a SOAP message.
+
+| | |
+|-|-|
+| **Category:**          | Http                                                      |
+| **Name:**              | `SoapMessage`                                         |
+| **Full Name:**         | `Cortex.DataTypes.Http.Soap.SoapMessage`     |
+| **Alias:**             | N/A                                                      |
+| **Description:**       | Any data type that is used to represent a SOAP message. |
+| **Default Value:**     | null                                                     |
```

Comment:
should the default values not be quoted with a backtick?

Reply:
Have added backtick to the default values :)